frontend/src/routes/ClusterManagement/Clusters/ImportCluster/ImportCluster.tsx

```diff
@@ -210,6 +210,12 @@ export function ImportClusterPageContent() {
                     <AcmSubmit
                         id="submit"
                         variant="primary"
+                        // isDisabled={
```

Comment:
Why Is this commented out? Should it be removed?

Reply:
These should not have been commented out. Was left in there from fixing unit tests. Has been corrected